source/dem/grid_motion.cc

```diff
@@ -77,7 +77,8 @@ GridMotion<dim, spacedim>::
       types::particle_index,
       std::map<types::boundary_id, particle_wall_contact_info_struct<spacedim>>>
       &particle_wall_pairs_in_contact,
-    const std::map<types::boundary_id, std::pair<Tensor<1, 3>, Point<3>>>
+    const typename DEM::dem_data_structures<
+      spacedim>::boundary_points_and_normal_vectors
```

Comment:
The key of that seems to be a global_face_id (see line 100) and not a boundary_id as define in the data_structures of boundary_points_and_normal_vectors. I know sometimes there's imaginary face id that are negative, is it not possible to have issues related to that?